usage/bigquery.go

```diff
@@ -166,6 +208,29 @@ func buildUsageSummaryQuery(table string, userID string, creatorID string, spec
 	return sql, args, nil
 }
 
+func buildTotalUsageSummaryQuery(table string, spec QuerySpec) (string, []interface{}, error) {
+
+	query := squirrel.Select(
```

Comment:
Wait, given there is no filter in this query, does it mean that we are returning usage from ALL users?? Shouldn't this API be admin-only at least? We shouldn't be exposing infrastructure-level metrics like this, right? What's the context for this? This PR is probably missing a description to answer all of these, I don't have much base for what to review here.

Reply:
Yes, this is a total usage for all users broken down in days to be used on the explorer! The original usage API is open, but you're right, that's the explorer one, not the studio one. So I am going to cover this with an admin check as well!